x/airdrop/types/errors.go

```diff
@@ -1,11 +1,63 @@
 package types
 
 import (
+	fmt "fmt"
+	"sort"
+	"strings"
+
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
 // x/airdrop module sentinel errors
 var (
 	ErrZoneDropNotFound    = sdkerrors.Register(ModuleName, 1, "zone airdrop not found")
 	ErrClaimRecordNotFound = sdkerrors.Register(ModuleName, 2, "claim record not found")
+	ErrUnknownStatus       = sdkerrors.Register(ModuleName, 3, "unknown status")
 )
+
+// NewMultiError returns an error aggregate using the given map.
+func NewMultiError(errors map[string]error) Errors {
```

Comment:
iirc, we have this in ICS already? Do we want to extract this out into the global `utils` package?

Reply:
Yes, I have it as a note in the commit message 204f417...
Though it would be best as a stand-alone micro lib imho